pkg/tools/renderer/helpers.go

```
@@ -0,0 +1,454 @@
+		return "HIGH"
+	}
+
+	if entry.CvssScore >= 7.0 || entry.EpssScore >= 0.3 {
+		return "MEDIUM"
+	}
+
+	return "LOW"
+}
+
+// getActionItems returns suggested action items for security professionals
+func getActionItems(entry *Entry) []string {
```

golangci-lint (1.64.8)

func `getActionItems` is unused

(unused)